modules/oxide/src/less/theme/globals/global.less

```diff
@@ -13,6 +13,7 @@
 @tinymce-border-radius: 0;
 @tinymce-border-width: 1px;
 @tinymce-box-shadow: none;
+@tinymce-inline-background-color: transparent;
```

Comment:
If we do keep the 2 variables, should this be `editor-header-inline-background-color` for consistency?

Reply:
Fixed!

Reply:
This doesn't look fixed? It's still starting with `tinymce`

Reply:
Now it's fixed.